apps/barista/src/components/live-example/live-example.ts

```diff
@@ -24,19 +24,24 @@ import {
   ViewContainerRef,
   ComponentRef,
   OnDestroy,
+  ViewEncapsulation,
 } from '@angular/core';
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
 import { EXAMPLES_MAP } from '@dynatrace/barista-components/examples';
+import * as hljs from 'highlight.js';
+import { BehaviorSubject, timer, Subscription } from 'rxjs';
 import { createComponent } from '../../utils/create-component';
-import { coerceBooleanProperty } from '@angular/cdk/coercion';
 
 @Component({
   selector: 'ba-live-example',
   templateUrl: 'live-example.html',
   styleUrls: ['live-example.scss'],
   host: {
+    class: 'ba-live-example',
     '[class.ba-live-example-dark]': 'themedark',
     '[class.ba-live-example-full-width]': 'fullwidth',
   },
+  encapsulation: ViewEncapsulation.None,
```

Comment:
I'm guessing you disabled the ViewEncapsulation to have the global stylesheet working, but would it also be possible to `@import` the highlight styles in the `live-example.scss` file and keep viewEncapsulation on?

Reply:
I disabled the view encapsulation to be able to style the code-lines I'm adding in the `_wrapCodeLines` function. Is there another way to do it better and with view encapsulation turned on?

Reply:
you can always break out of the view encapsulation with the `:host` attribute but can you provide a simple example?

Reply:
@lukasholzer Not sure what you mean with breaking out the view encapsulation via the `:host` attribute. `:host` is there for accessing the host element, not for breaking out the encapsulation. I guess you mean `ng-deep`?

Reply:
:ng-deep it was – thanks.